panels/dev-service/ha-panel-dev-service.html

```diff
@@ -49,7 +49,7 @@
       }
 
       .attributes tr:nth-child(even) {
-        background-color: #eee;
+        background-color: var(--paper-listbox-background-color,#eee)
```

Comment:
I agree.

We should have hierarchical variable declarations.

 - Colors: ie `--google-grey-200`, red, blue etc
 - Brand variables: ie `--primary-color`, `--accent-color`
 - Applied colors: `--table-row-alternative-background-color`, `--table-row-background-color`

Only the last two categories should be used to style things. We should also make sure that users don't have to override variables by Google like `--paper-listbox-background-color`.